acq4/devices/DAQPressureControl/pressurecontrol.py

```diff
@@ -1,10 +1,18 @@
 from __future__ import print_function
-from acq4.util import Qt
+
 from ..PressureControl import PressureControl
 
 
 class DAQPressureControl(PressureControl):
     """Pressure control device driven by DAQ analog/digital channels.
+    The configuration for these devices should look like:
```

Comment:
For rst rendering, use:

    The configuration for these devices should look like::
    
        sources: